src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -914,4 +931,163 @@ else if(include.get(i).equals(SERVICEPOINTS_INCLUDE)) {
     });
   }
 
+  /**
+   *
+   * @param locateUserFields - a list of fields to be used for search
+   * @param value - a value to search
+   * @return
+   */
+  private String buildQuery(List<String> locateUserFields, String value) {
+    return locateUserFields.stream()
+      .map(field -> new StringBuilder(field).append("==\"").append(value).append("\"").toString())
+      .collect(Collectors.joining(" or "));
+  }
+
+  /**
+   * Maps aliases to configuration parameters
+   * @param fieldAliasList - a list of aliases
+   * @return a list of user fields to use for search
+   */
+  private io.vertx.core.Future<List<String>> getLocateUserFields(List<String> fieldAliasList, java.util.Map<String, String> okapiHeaders) {
+    //TODO:
+
+    String okapiURL = okapiHeaders.get(OKAPI_URL_HEADER);
+    String tenant = okapiHeaders.get(OKAPI_TENANT_HEADER);
+    String token = okapiHeaders.get(OKAPI_TOKEN_HEADER);
+
+    Matcher matcher = HOST_PORT_PATTERN.matcher(okapiURL);
+    if (!matcher.find()) {
+      return io.vertx.core.Future.failedFuture("Could not parse okapiURL: " + okapiURL);
+    }
+    io.vertx.core.Future<List<String>> future = io.vertx.core.Future.future();
+
+    String host = matcher.group(1);
+    String port = matcher.group(2);
+
+    ConfigurationsClient configurationsClient = new ConfigurationsClient(host, StringUtils.isNotBlank(port) ? Integer.valueOf(port) : DEFAULT_PORT, tenant, token);
+    StringBuilder query = new StringBuilder("module==USERSBL AND (")
+      .append(fieldAliasList.stream().map(f -> new StringBuilder("code==").append(f)).collect(Collectors.joining(" or ")))
```

Comment:
`f` needs to be quoted/escaped if it contains whitespace or any of there characters: ( ) = <> " /
See https://www.loc.gov/standards/sru/cql/spec.html#bnf

Reply:
Done